python/glchat-sdk/glchat_sdk/message.py

```diff
@@ -183,16 +187,17 @@ def _prepare_files(
     def _make_streaming_request(
         self,
         url: str,
-        data: Dict[str, Any],
-        files: List[Tuple[str, Tuple[str, Union[str, BinaryIO, bytes], str]]] | None,
-        headers: Dict[str, str],
+        data: dict[str, Any],
+        files: list[tuple[str, tuple[str, str | BinaryIO | bytes, str]]] | None,
+        headers: dict[str, str],
     ) -> Iterator[bytes]:
         """Make the streaming HTTP request.
 
         Args:
             url (str): API endpoint URL
             data (Dict[str, Any]): Request data
-            files (List[Tuple[str, Tuple[str, Union[str, BinaryIO, bytes], str]]] | None): Prepared files data
+            files (List[Tuple[str, Tuple[str, Union[str, BinaryIO, bytes], str]]]
+                | None, Optional): Prepared files data
```

Comment:
```suggestion
            files (List[Tuple[str, Tuple[str, Union[str, BinaryIO, bytes], str]]]
                | None): Prepared files data
```
`files` is not optional as it does not have any fallback values on `_make_streaming_request`